app/views/samples/index.haml

```diff
@@ -36,7 +36,7 @@
                 %td
                   =check_box_tag 'sample_ids[]', sample.id, false, { id: "sample_ids.#{sample.id}" }
                   %label.row{for: "sample_ids.#{sample.id}"}
-                %td= sample.uuid
+                = short_uuid_with_title :td, sample.uuid
```

Comment:
I don't think it's a good idea to merge the UUID element with the enclosing table cell. This doesn't offer much flexibility and could easily lead to conflicts between styles that apply to a UUID display and the enclosing element. They're separate concerns. As mentioned before, I think `<abbr>` would be the best element type for short UUIDs. And that could be hard-coded into `short_uuid_with_title`.

Reply:
I see what you mean. I'm doing the following now:

`%td= short_uuid_with_title :span, sample.uuid`

And it looks exactly as before_

![image](https://user-images.githubusercontent.com/13782680/158366410-d529eb1a-e863-43f2-aaac-10a862366390.png)

While for `:abbr` an underline appears:

![image](https://user-images.githubusercontent.com/13782680/158366221-4c20dcd0-1407-4130-b068-6c9669068ead.png)

I should proceed with `<span>` then?

Reply:
I would still use `abbr` because it's the correct semantics. We can easily adjust the visuals for `abbr`.

Reply:
I commited it using abbr. I let there the html tag on the helper as a parameter for flexibility... Just to be sure it makes sense: should be always abbr and remove the parameter name in the helper:

In other words, now we have this:
```
def short_uuid_with_title(name, uuid)
    content_tag(name, format_uuid(uuid), title: uuid)
  end
```

We should change it to this:
def short_uuid_with_title(uuid)
    content_tag(:abbr, format_uuid(uuid), title: uuid)
  end
```
?

Reply:
Yes, I'd always use `abbr`. The other alternative is not really flexibly, because only the element name can be customized, but not its attributes. That would even require more effort, but it's simply not necessary nor helpfule.
The combination `abbr` + short UUID + title is a functional unit. There's no need for any other element type. It can easily be wrapped in anything else (like `td`).